Common/PM_datatypes.hpp

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+// please change this name
```

Comment:
WaypointType was originally supposed to be used in PM only. If we no longer need it in the PM -> AM struct then it can be removed